Sources/SwiftInspectorCore/InitializerAnalyzer.swift

```diff
@@ -192,19 +192,19 @@ private final class InitializerSyntaxReader: SyntaxVisitor {
     self.onNodeVisit = onNodeVisit
   }
 
-  func visit(_ node: ClassDeclSyntax) -> SyntaxVisitorContinueKind {
+  override func visit(_ node: ClassDeclSyntax) -> SyntaxVisitorContinueKind {
```

Comment:
So it's a subclass and not a protocol?

Reply:
Yes, this was changed with the new version https://github.com/apple/swift-syntax/blob/master/Changelog.md#swift-52

Reply:
Cool release notes are fairly helpful